models/eori-validation.ts

```diff
@@ -20,7 +20,7 @@ export const getEORIValidation = async (
     const siret = verifySiret(eori);
     const data = await clientEORI(siret);
     if (!data) {
-      throw new Error('EOS response is empty');
+      return { eori, isValid: false };
```

Comment:
I don't get why this is necessary. A numero not valid means that the API responded. This ensure that a incorrect response from API will be logged into sentry and will result in a `APINotResponding` error.

Reply:
Makes sense